backend/telemetry_core/tests/soak_tests.rs

```diff
@@ -265,7 +266,7 @@ fn chain_names(total: usize) -> impl Iterator<Item = String> {
 /// General arguments that are used to start a soak test. Run `soak_test` as
 /// instructed by its documentation for full control over what is ran, or run
 /// preconfigured variants.
-#[derive(StructOpt, Debug)]
```

Comment:
why removed?

Reply:
Oh I got a compiler warning that it wasn't used (presumably because the error type isn't exported and we don't debug it anywhere here). Must be a newer warning :)